source/agora/tool/main.d

```diff
@@ -0,0 +1,98 @@
+/+
+ dub.sdl:
+ name "sometest"
+ dependency "vibe-d" version="~>0.8"
++/
+
```

Comment:
@AndrejMitrovic : I was thinking of moving this to its own repository, using submodules for the Agora dependency, and I think everything would *just work*. There would be a need to update the Agora submodule from time to time but it wouldn't put any pressure on Agora development. Thoughts ?

Reply:
Yes I like this too.